src/main/resources-filtered/META-INF/maven/archetype-metadata.xml

```diff
@@ -27,6 +27,9 @@
     <requiredProperty key="spring-boot-version">
```

Comment:
same here, I don't think we need to specify spring-boot-version anymore

Reply:
"_required_" doesn't mean that the user should necessarily specify the parameter. That's managed from a default value, but obviously, we can delete